Source/WebKit/Shared/ios/NativeWebTouchEventIOS.mm

```diff
@@ -113,26 +107,25 @@ static CGFloat radiusForTouchPoint(const _UIWebTouchPoint& touchPoint)
         auto radius = radiusForTouchPoint(touchPoint);
         platformTouchPoint.setRadiusX(radius);
```

Comment:
This feels like really dumb questions, but where is platformTouchPoint defined?? I don't see it.

Reply:
It's defined a few lines above (NativeWebTouchEventIOS.mm:105)